templates/search_results.html

```diff
@@ -8,47 +8,36 @@
 </head>
 
 <body>
-	<h1>Welcome to the Search Results Page! :)</h1>
-
-	<form class="search-form" action="/" method="POST">
-		<div class="topnav">
-			<div class="row">
-				<div class="col-9">
-					<input type="text" placeholder="Search.." class="form-control">
-				</div>
-				<div class="col-2">
```

Comment:
thank you, thank you, thank you for getting rid of this! lol we did not need her (col) here!